README.rst

```diff
@@ -84,7 +84,9 @@ Using the API
     from google.cloud import logging_v2
 
     client = logging_v2.LoggingServiceV2Client()
-    entries = []
```

Comment:
Though this PR only changed the issue that the list `entries` is empty, I think we need to add how to set `resources` and `log_name`. What do you think?

Reply:
Yes, that would be useful to have too.

Reply:
Thanks, I added the note.